src/main/java/Duke.java

```diff
@@ -1,10 +1,129 @@
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    static ArrayList<Task> storage;
+    static ArrayList<String> instructions = new ArrayList<String>();
+    static int count = 0;
+    private static Data data;
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        
+        Scanner sc = new Scanner(System.in);
+        storage = new ArrayList<Task>();
+        data = new Data("./data/duke.txt");
+        data.createFolder();
+        data.createFile();
+        data.loadTasks();
+        // instruction for the chatbot to follow
+        String instruction;
+        System.out.println("Hello! I'm Area.\n" +
+                "What can I do for you?\n");
+
+        boolean chatting = true;
```

Comment:
Perhaps you could change the variable name to sound like a boolean to align better with naming practices
```suggestion
        boolean isChatting = true;
```

Reply:
thanks